README.md

```diff
@@ -1,13 +1,46 @@
 # SciHive 
 
-Read, collaborate and talk about scientific papers.
+Read, collaborate and talk about scientific papers
 
-README coming soon
+# Set Up
 
-###Notes
-- Please install pandoc to support references extraction from here: https://pandoc.org/installing.html
+### Prerequisites 
+- You will need both python and pip on your machine to run the back-end code.  
+To install pip for python3 execute the following:  
+`sudo apt-get install python3-pip`
+
+*It is reccommended to use a virtual environment when installing the various necessary modules.*  
+You can obtain virtualenv with a simple pip3 install.
+To start your virtual environment run the following code in the terminal:  
+```python
+cd scihive-backend
+source venv/bin/activate
```

Comment:
Are you using python-Anaconda? I didn't know that `source` works with vanilla python